src/test/java/uk/gov/ons/census/casesvc/service/ReceiptProcessorTest.java

```diff
@@ -1,72 +1,80 @@
 package uk.gov.ons.census.casesvc.service;
 
-import static org.mockito.Mockito.*;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 import static uk.gov.ons.census.casesvc.service.ReceiptProcessor.QID_RECEIPTED;
+import static uk.gov.ons.census.casesvc.testutil.DataUtils.getTestResponseManagementEvent;
 
-import java.time.OffsetDateTime;
-import java.util.HashMap;
-import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
 import org.jeasy.random.EasyRandom;
 import org.junit.Test;
-import uk.gov.ons.census.casesvc.model.dto.PayloadDTO;
-import uk.gov.ons.census.casesvc.model.dto.Receipt;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.ons.census.casesvc.logging.EventLogger;
+import uk.gov.ons.census.casesvc.model.dto.ReceiptDTO;
+import uk.gov.ons.census.casesvc.model.dto.ResponseManagementEvent;
 import uk.gov.ons.census.casesvc.model.entity.Case;
 import uk.gov.ons.census.casesvc.model.entity.EventType;
 import uk.gov.ons.census.casesvc.model.entity.UacQidLink;
 import uk.gov.ons.census.casesvc.model.repository.CaseRepository;
 import uk.gov.ons.census.casesvc.model.repository.UacQidLinkRepository;
 
+@RunWith(MockitoJUnitRunner.class)
 public class ReceiptProcessorTest {
   private static final UUID TEST_CASE_ID = UUID.randomUUID();
-  private static final String TEST_QID = "test_qid";
-  private static final String TEST_UAC = "test_uac";
+  private static final String TEST_QUESTIONNAIRE_ID = "123";
+
+  @Mock private CaseProcessor caseProcessor;
+
+  @Mock private UacQidLinkRepository uacQidLinkRepository;
+
+  @Mock private CaseRepository caseRepository;
+
+  @Mock private UacProcessor uacProcessor;
+
+  @Mock private EventLogger eventLogger;
+
+  @InjectMocks UacProcessor underTest;
```

Comment:
UAcProcessor isn't underTest (generic name anyway).  It's also redeclared locally in each Test method